configs/TAKERF411/config.h

```diff
@@ -0,0 +1,104 @@
+/*
+ * This file is part of Betaflight.
+ *
+ * Betaflight is free software. You can redistribute this software
+ * and/or modify this software under the terms of the GNU General
+ * Public License as published by the Free Software Foundation,
+ * either version 3 of the License, or (at your option) any later
+ * version.
+ *
+ * Betaflight is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ *
+ * See the GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public
+ * License along with this software.
+ *
+ * If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#pragma once
+
+#define FC_TARGET_MCU                   STM32F411
+
+#define BOARD_NAME                      TAKERF411
+#define MANUFACTURER_ID                 GEPRC
```

Comment:
```suggestion
#define MANUFACTURER_ID                 GEPR
```

Reply:
The TAKERF411/config.h   is  corresponds to unified-targets\configs\default\GEPRC-TAKERF411.config。if change config.h's MANUFACTURER_ID (from GEPRC to GEPR). Will it cause other problems?

Reply:
GEPR is the manufacturer ID.

https://github.com/betaflight/unified-targets/blob/master/Manufacturers.md